patterns/blue-green-upgrade/environment/versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 1.0"
+  required_version = ">= 1.4.0"
```

Comment:
are we using any features that require a MSV of 1.4?

Reply:
the required version should be the minimum supported version - so unless we are using a feature thats only available starting with Terraform 1.4, then we default to 1.0

Reply:
ok